src/main/java/thePackmaster/screens/PackSetupScreen.java

```diff
@@ -101,6 +103,7 @@ public void open(int randomPacks, int packChoices) {
         rng = new Random(Settings.seed);
 
         packPool.clear();
+        MetricsPatches.packChoices.clear();
```

Comment:
There's one other place that we should clear this list -- in `OpeningRunScreenPatch.SetTheThing`, in the if branch for all packs mode. That branch never opens up the pack selection screen, and as currently written it would leave the pack choices data the same as the previous packmaster run (e.g. if you did a normal run, then turned on all packs mode and did another run).

Now that I think about it, this might be another case where you could make one clear call in `SpireAnniversary5Mod.receivePreStartGame` instead. That works for `currentPoolPacks`, so it should work for this.